plugins/nvm/nvm.plugin.sh

```diff
@@ -1,12 +1,68 @@
 #! bash oh-my-bash.module
+# Description: automatically load nvm 
+#
+# @var[opt] OMB_PLUGIN_NVM_AUTO_USE enable .nvmrc autoload
 
 # Set NVM_DIR if it isn't already defined
 [[ -z "$NVM_DIR" ]] && export NVM_DIR="$HOME/.nvm"
 
 
 # Try to load nvm only if command not already available
 if ! _omb_util_command_exists nvm; then
-  [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"  # This loads nvm
-  # This is done as part of completions!!!
-  # [ -s "$NVM_DIR/bash_completion" ] && \. "$NVM_DIR/bash_completion"  # This loads nvm bash_completion
+  # shellcheck source=/dev/null
+  [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
+fi
+
+# Optional .nvmrc autoload
+if _omb_util_command_exists nvm && [[ ${OMB_PLUGIN_NVM_AUTO_USE} == true ]]; then
+  find-up () {
+    path=$(pwd)
+    while [[ "$path" != "" && ! -e "$path/$1" ]]; do
+        path=${path%/*}
+    done
+    echo "$path"
+  }
+  
+  cdnvm(){
+    cd "$@" || return $?
```

Comment:
```suggestion
    cd "$@" || return "$?"
```

In case `IFS` has an unusual value like `IFS=0`.